client/signup/steps/site-or-domain/index.jsx

```diff
@@ -1,3 +1,5 @@
+// eslint-disable-next-line import/named
+import { SelectItems } from '@automattic/intent-screen';
```

Comment:
Add to disable because it was complaining:
> SelectItems not found in '@automattic/intent-screen' eslint([import/named](https://github.com/import-js/eslint-plugin-import/blob/v2.25.4/docs/rules/named.md))
Trying to fix that